packages/block-library/src/details/save.js

```diff
@@ -4,12 +4,12 @@
 import { RichText, useBlockProps, InnerBlocks } from '@wordpress/block-editor';
 
 export default function save( { attributes } ) {
-	const { showContent } = attributes;
+	const { name, showContent } = attributes;
 	const summary = attributes.summary ? attributes.summary : 'Details';
 	const blockProps = useBlockProps.save();
 
 	return (
-		<details { ...blockProps } open={ showContent }>
+		<details { ...blockProps } name={ name } open={ showContent }>
```

Comment:
Won't this cause deprecation issues? Because the `name` attribute is added with an empty string if the value is empty. 

I think either it should be omitted entirely, and then it also won't be an issue for deprecations, or we need to add a proper deprecation in place.

Reply:
I changed it to `name={ name || null }` so this is only rendered, if name has a value.